.github/workflows/publish-jvm.yml

```diff
@@ -190,7 +190,7 @@ jobs:
             echo "PUB_MODE=-PSNAPSHOT" >> $GITHUB_ENV
           fi
 
-      - if: ${{ github.event.inputs.github_publish == 'true' }}
+      - if: ${{ github.event.inputs.github_publish == 'true' || inputs.github_publish == true }}
```

Comment:
I think it's enough to use only `inputs.ghithub_publish` instead of the inputs from the `github.event` context

Reply:
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs
Yea you may be right, I thought that
- workflow_call → boolean inputs under `inputs.<name>`
- workflow_dispatch → string inputs under `github.event.inputs.<name>`

But, the documentation I linked suggests it works as you say. I'll change that.